### PR TITLE
Update development instructions for local development

### PR DESCRIPTION
**Pull request by Stevenjin8**

### CODE REVIEW

File: Development.md [hunk 1]
```diff
@@ -23,8 +23,19 @@ There are a variety of config options that can be used to replace components wit
 * `FAKE_CA="true"`: this will use self-signed fake certificates, eliminating a dependency on a CA
 * `XDS_ADDRESS=""`: disables XDS client completely
 * `LOCAL_XDS_PATH=./examples/localhost.yaml`: read XDS config from a file.
-
-Together, `FAKE_CA="true" XDS_ADDRESS="" LOCAL_XDS_PATH=./examples/localhost.yaml cargo run --features testing` (with `--no-default-features` if you have FIPS disabled) can be used to run entirely locally, without a Kubernetes or Istiod dependency.
+* `CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUNNER="sudo -E"`: have cargo run as sudo
+* `PROXY_MODE=dedicated`: Dedicated mode is better for local development as it works for 95% of cases.
```
Comment (bleggett) [inline Development.md:27]:
```suggestion
* `PROXY_MODE=dedicated`: Dedicated mode is the single-tenant proxy mode and is strongly recommended for local development, as it works for 95% of cases and doesn't require manually constructing Linux network namespaces to use.
```

File: Development.md [hunk 2]
```diff
@@ -23,8 +23,19 @@ There are a variety of config options that can be used to replace components wit
...
+
+Together, the following command (with `--no-default-features` if you have FIPS disabled) can be used to run entirely locally, without a Kubernetes or Istiod dependency.
```
Comment (bleggett) [inline Development.md:29]:
```suggestion
The following command (with `--no-default-features` if you have FIPS disabled) can be used to run entirely locally, without a Kubernetes or Istiod dependency.
```

**Comment by Stevenjin8** [comment]
@bleggett Thanks for the suggestion. TBH I'm still catching up on what in-pod.

**Review by bleggett — APPROVED** [review]
thanks!

**Review by howardjohn — APPROVED** [review]

**Pull request merged**